src/extension/src/coreTemplateStudio.ts

```diff
@@ -46,24 +46,23 @@ export class CoreTemplateStudio {
     if (context) {
       extensionPath = context.extensionPath;
     } else {
-      extensionPath = path.join(__dirname, '..');
+      extensionPath = path.join(__dirname, "..");
     }
 
     if (platform === CONSTANTS.API.WINDOWS_PLATFORM_VERSION) {
       executableName += ".exe";
     }
 
-    let apiPath = vscode.Uri.file(
-      path.join(extensionPath, "src", "api", platform, executableName)
-    ).fsPath;
-
-    let apiWorkingDirectory = path.join(
+    let apiPath = path.join(
```

Comment:
What is `vscode.Uri.file`? Has it been removed intentionally?

Reply:
I actually removed it for testing purposes since I couldnt use the vs code module because the test runs without running the extension. I forgot to add it back - thanks for reminding me! I just added it back

Reply:
`path.join` takes care of the path for us. We need to remove dependencies on `vscode` here so we can run the generate script by itself, outside the extension.